### PR TITLE
Update Atom Syntax to Atom 1.0 API

### PR DESCRIPTION
**Pull request by arminrosu**
Ahoy!

TLDR; Turn this:

screen shot 2016-04-04 at 15 47 20

to this:

screen shot 2016-04-04 at 15 48 23

Notice how the UI changes with the theme.

@daylerees could you please have a look at these:

* _build/resources/patterns/atom/syntax-variables.pattern_ contains more variables that can be set in Atom. I don't know which corresponds to your own variables. From what I can tell now, the atom syntax looks great anyway, but this would add some polish.
* how do _you_ build the project? I ran `cd ./build; composer install; php run.php raincolour;`.

I will commit the compiled files too if you think the PR is good.

Thanks for making pretty colors for me to look at :)

### CODE REVIEW

**Comment by daylerees** [comment]
Thanks for this @arminrosu , gonna take a proper look this weekend! You're right about the build process, and don't worry about excess values. You just have to match up the theme as best as possible. Don't need to use every value. :)

**Comment by daylerees** [comment]
Thanks!

**Comment by jonboiser** [comment]
> I will commit the compiled files too if you think the PR is good.

Does the ./atom folder need to be rebuilt? Last update to that folder is still reading "2 years ago".

**Comment by arminrosu** [comment]
Yes it does. I emailed Dayle about it a few days ago, waiting for a response.

**Comment by cusxio** [comment]
I made a [temporary repository](https://github.com/cusxio/atom-colour-schemes) that holds the rebuilt colour schemes for Atom so that other users don't have to rebuild them.

**Comment by arminrosu** [comment]
@cusxio thanks, great idea!
